bin/sync_slurm_acct.py

```diff
@@ -39,6 +39,8 @@
 SYNC_TIMESTAMP_FILENAME = "/var/cache/%s.timestamp" % (NAGIOS_HEADER)
 SYNC_SLURM_ACCT_LOGFILE = "/var/log/%s.log" % (NAGIOS_HEADER)
 
+MAX_USERS_JOB_CANCEL = 30
```

Comment:
ok, now these are true users; so you can lower it again to 10 or so? not sure what a good limit is here

Reply:
neither am I :-)